File: WorkIt/app/src/main/java/prot3ct/workit/views/list_task_requests/ListTaskRequestsActivity.java
package prot3ct.workit.views.list_task_requests;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import prot3ct.workit.R;
import prot3ct.workit.views.list_task_requests.base.ListTaskRequestContract;
import prot3ct.workit.views.navigation.DrawerUtil;

public class ListTaskRequestsActivity extends AppCompatActivity {
    private ListTaskRequestContract.Presenter presenter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_list_task_requests);

        ListTaskRequestsFragment listTaskRequestsFragment =
                (ListTaskRequestsFragment) getSupportFragmentManager().findFragmentById(R.id.fragment_container);

        if (listTaskRequestsFragment == null) {
            listTaskRequestsFragment = ListTaskRequestsFragment.newInstance();

            getSupportFragmentManager()
                    .beginTransaction()
                    .add(R.id.fragment_container, listTaskRequestsFragment)
                    .commit();
        }

        this.presenter = new ListTaskRequestsPresenter(listTaskRequestsFragment, this);
        listTaskRequestsFragment.setPresenter(this.presenter);
    }
}

File: WorkIt/app/src/main/java/prot3ct/workit/views/list_task_requests/ListTaskRequestsActivity.kt
package prot3ct.workit.views.list_task_requests

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import prot3ct.workit.R
import prot3ct.workit.views.list_task_requests.ListTaskRequestsFragment
import prot3ct.workit.views.list_task_requests.ListTaskRequestsPresenter
import prot3ct.workit.views.list_task_requests.base.ListTaskRequestContract

class ListTaskRequestsActivity : AppCompatActivity() {

    private lateinit var presenter: ListTaskRequestContract.Presenter

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_list_task_requests)
        var listTaskRequestsFragment =
            supportFragmentManager.findFragmentById(R.id.fragment_container) as ListTaskRequestsFragment?
        if (listTaskRequestsFragment == null) {
            listTaskRequestsFragment = ListTaskRequestsFragment.newInstance()
            supportFragmentManager
                .beginTransaction()
                .add(R.id.fragment_container, listTaskRequestsFragment)
                .commit()
        }
        presenter = ListTaskRequestsPresenter(listTaskRequestsFragment, this)
        listTaskRequestsFragment.setPresenter(presenter)
    }
}